JPS_Ontology/ontology/ontobuiltenv/OntoBuiltEnv.csv

```diff
@@ -121,7 +121,9 @@ Pharmacy,Class,Medical,IS-A,,,,A healthcare facility where medicinal drugs are p
 BuildingFloors,Class,,,Property,,,A concept to describe amount of floors in a building.,https://www.theworldavatar.com/kg/ontobuiltenv/,Building floors
```

Comment:
As commented previously, I strongly recommend renaming `BuildingFloors` to `NumberOfFloors` (or similar, making it clear that this class only refers to a number, and not the floors themselves). In addition, in the comment, "amount" should be replaced with "number".

Reply:
ok changed in the latest revision